src/components/events-carrosel/index.js

```diff
@@ -48,44 +44,49 @@ const Events = () => {
         breakpoints={{
           '@0.00': {
             slidesPerView: 1,
-            spaceBetween: 1
+            spaceBetween: 1,
           },
           '@0.75': {
             slidesPerView: 2,
-            spaceBetween: 2
+            spaceBetween: 2,
           },
           '@1.00': {
             slidesPerView: 3,
-            spaceBetween: 3
-          }
+            spaceBetween: 3,
+          },
         }}
         modules={[Navigation]}
         className="mySwiper"
       >
         <section>
           <div className="swiper-slide">
             <ul>
-              {attributesEvents.map((events, key) =>
+              {attributesEvents.map((events, key) => (
                 <li key={key}>
                   <SwiperSlide>
                     <div>
                       <div>
-                        <img className="img-foto" src={urlCms + events.imagem_url} />
+                        <img
+                          className="img-foto"
```

Comment:
Consistência entre idiomas, essa é só uma ideia de nome:

```suggestion
                          className="event-image"
```

Reply:
Alteração feita.